.github/workflows/lint.yml

```diff
@@ -9,4 +9,4 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
       - run: yarn install --frozen-lockfile
-      - run: yarn run lint-workflow
+      - run: yarn run lint:workflow
```

Comment:
Is this part of the storybook?

Reply:
I just changed some of the scripts to keep the style—not strictly a part of storybook but I didn't think it was that big of a fix that necessitated a whole new branch